spec/centos_spec.rb

```diff
@@ -80,13 +80,10 @@
 
     it 'Install os shipped package' do
       expect(chef_run).to install_package('mariadb-server')
+      expect(os_package).to notify('directory[/var/log/mysql]').to(:create).immediately
       expect(os_package).to notify('service[mysql]').to(:start).immediately
     end
 
-    it 'Create Log directory' do
```

Comment:
I am not quite sure how to add .with(...) with specific attribute here. This directory is declared as 'nothing'. If we continue to use 'create_directory(...).with(...)', it will fail the case. We cannot add .with(...) to notify matcher here, because notification_matcher does not provide 'with' either.